tests/test_break_continue_colors.py

```
@@ -0,0 +1,43 @@
+    continue_color = theme["continue"] # lightgray
+    box_color = theme["box"]          # lightyellow
+    
+    graph = generate_graphviz_flowchart(code, style_config=theme)
+    dot_source = graph.source
+    
+    print("Checking for break color:", break_color)
+    if f'fillcolor="{break_color}"' in dot_source or f'fillcolor={break_color}' in dot_source:
+        print("PASS: Break node has correct color.")
+    else:
+        print(f"FAIL: Break node missing color {break_color}")
+        raise AssertionError("Break node styling failed")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)